packages/components/MIGRATING.md

```diff
@@ -162,6 +162,108 @@ return (
+
+-   The styling / layout is now controlled on the subcomponent `Dialog.Content`
+
+-   The subcomponent `Dialog.Content` is used to display the dialog container.
+    It provides no styling by default and can be used to wrap a custom content.
+
+    -   `roundedCorners` is now called `rounded` and is a boolean.
+
+    -   The `width` and `maxHeight` props were removed. The Flyout container adjusts to the content inside. You can use a custom component inside if neccessary. you can use `minWidth`, `maxWidth` and `minHeight` to override the default values.
+
+    -   The `strategy` and `role` props were removed to simplify the API.
+
+    -   The `enablePortal` and `verticalAlignment` props were removed to standardize the dialog.
+
+        The dialog now uses a portal by default and is placed centered
+
+-   The subcomponents `Dialog.Header`, `Dialog.Body`, `Dialog.Footer` and `Dialog.SideContent` can be used to add defaut styling to the content inside of `Dialog.Content`.
+
+#### Old
+
+```tsx
+const [isOpen, setIsOpen] = useState(false);
+
+return (
+    <Button icon={<IconIcon/>} ref={triggerRef} onClick={() => setIsOpen(!isOpen)}>
+        Dialog Trigger
+    </Button>
+
+    <Dialog {...args} anchor={triggerRef} open={isOpen} handleClose={() => setIsOpen(false)}>
+        <DialogBody padding="spacious">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad illum impedit iure numquam praesentium vel.
+                Distinctio perferendis, suscipit! Dolor doloremque et ex, modi nobis officiis perspiciatis quis tempora
+                temporibus voluptates?
+            </p>
+        </DialogBody>
+        <DialogFooter
+            actionButtons={[
+                {
+                    children: 'Close',
+                    onClick: () => setIsOpen(false),
+                },
+            ]}
+        ></DialogFooter>
+    </Dialog>
+)
+```
+
+#### New
+
+```tsx
+<Dialog.Root>
+    {/* Pass in a Trigger component */}
+    <Dialog.Trigger>
+        <Button>Open dialog</Button>
+    </Dialog.Trigger>
+    {/* Pass in the Dialog Content component */}
+    <Dialog.Content {...args}>
+        {/* Pass in the Dialog SideContent component */}
+        <Dialog.SideContent>
+            <div className="tw-bg-box-positive-strong tw-h-full tw-w-full"></div>
+        </Dialog.SideContent>
+        {/* Pass in the Dialog Header component */}
+        <Dialog.Header>Header</Dialog.Header>
+        {/* Pass in the Dialog Body component */}
+        <Dialog.Body {...args} />
+        {/* Pass in the Dialog Footer component */}
+        <Dialog.Footer>
+            <div className="tw-flex tw-h-fit tw-justify-end tw-gap-2">
+                <Button emphasis="default">Cancel</Button>
+                <Button>Submit</Button>
+            </div>
```

Comment:
Not a 1-to-1 with the old code, please make sure it is

Reply:
adjusted